our_site/planner/forms.py

```diff
@@ -2,6 +2,20 @@
 from django.core.exceptions import ValidationError
 
 from django.contrib.auth.forms import User
+from .models import Event, Profile, Role
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+class RoleForm(forms.ModelForm):
```

Comment:
Correct me if I'm wrong, but inheriting from forms.ModelForm allows for the automatic generation of all the desired fields (or at least, you can specify which fields you want) required for a role, no (minus the excluded ones)? If so, why do we have manually generated fields for name, description, and amount? I might just not understand this one well, so sorry if that's wrong.

I think a problem here is that using this allows a user to add roles to *any* event they want, not just events they own, which seems like a problem.